packages/core/src/telemetry/types.ts

```diff
@@ -246,6 +246,23 @@ export class FlashFallbackEvent {
   }
 }
 
+export enum LoopType {
+  TOOL_CALL = 'tool_call',
```

Comment:
We're going to have a number of tool call type loops. We should make this a bit more specific. Ditto with the Content one below